lib/luaxtable.c

```diff
@@ -209,11 +327,15 @@ LUNATIK_NEWLIB(xtable, luaxtable_lib, &luaxtable_class, luanetfilter_flags);
 
 static int __init luaxtable_init(void)
 {
+	luaxtable_hooks.match = luarcu_newtable(LUARCU_DEFAULT_SIZE, false);
```

Comment:
Actually, this can fail (e.g., return NULL), in this case you should return an error condition.. probably `-ENOMEM`